docs/Biblioteca_Asersiones_Y_Test_Runner.md

```diff
@@ -0,0 +1,33 @@
+# Biblioteca asersiones

```

Comment:
El nombre del fichero sigue siendo erróneo. Os agradecería de todas formas que pusiérais en cada objetivo sólo lo necesario para el mismo. Si tenéis que hacerlo con otro sin fusionar, haced un PR a esa rama.

Reply:
Si, estos errores y el anterior comentado se deben a sacar esta rama el objetivo 4 que estaba erroneo. Lo cambio como me indica para que salga solo lo referente a este objetivo.